### PR TITLE
Localization for Language of Simplified Chinese (zh-cn)  

### PR DESCRIPTION
**Pull request by Li-Yanzhi**
Also with a fix of mismatch of name & text in english version.  (Couldn't find a way to create a seperate PR for each change. )

### CODE REVIEW

File: hockeysdk/src/main/res/values-zh/strings.xml [hunk 1]
```diff
@@ -0,0 +1,94 @@
+<resources>
+    <!-- Crash Dialog -->
+    <string name="hockeyapp_crash_dialog_app_name_fallback">此应用</string>
+    <string name="hockeyapp_crash_dialog_title">%s 意外终止</string>
+    <string name="hockeyapp_crash_dialog_message">您愿意发送一份匿名报告以便我们修复问题？</string>
```
Comment (DennisPan) [inline hockeysdk/src/main/res/values-zh/strings.xml]: Should have `吗` at the end?

File: hockeysdk/src/main/res/values-zh/strings.xml [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    <!-- Download Failed -->
+    <string name="hockeyapp_download_failed_dialog_title">下载失败</string>
+    <string name="hockeyapp_download_failed_dialog_message">无法下载更新，是否重试？</string>
+    <string name="hockeyapp_download_failed_dialog_negative_button">取消</string>
+    <string name="hockeyapp_download_failed_dialog_positive_button">重试</string>
+
+    <!-- Update -->
+    <string name="hockeyapp_update_mandatory_toast">%s 有更新可用并且为强制更新! </string>
+    <string name="hockeyapp_update_dialog_title">有更新</string>
+    <string name="hockeyapp_update_button">更用</string>
```
Comment (DennisPan) [inline hockeysdk/src/main/res/values-zh/strings.xml]: `更新`？

File: hockeysdk/src/main/res/values-zh/strings.xml [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    <string name="hockeyapp_update_dialog_message">显示关于更新的信息？</string>
```
Comment (DennisPan) [inline hockeysdk/src/main/res/values-zh/strings.xml]: `显示更新信息？`?

File: hockeysdk/src/main/res/values-zh/strings.xml [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+    <string name="hockeyapp_update_dialog_negative_button">忽略</string>
+    <string name="hockeyapp_update_dialog_positive_button">显示</string>
+
+    <!-- Expiry Info -->
+    <string name="hockeyapp_expiry_info_title">版本过期</string>
+    <string name="hockeyapp_expiry_info_text">%s 已经过期，请在 HockeyApp 中检查是否存在更新版本。</string>
```
Comment (DennisPan) [inline hockeysdk/src/main/res/values-zh/strings.xml]: ｀ ，` --> `。`

`检查` --> `查看`？

File: hockeysdk/src/main/res/values-zh/strings.xml [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+    <string name="hockeyapp_feedback_failed_title">挂起</string>
+    <string name="hockeyapp_feedback_failed_text">您是否想再次发送您的反馈？</string>
+    <string name="hockeyapp_feedback_name_hint">姓名</string>
+    <string name="hockeyapp_feedback_email_hint">电子邮件</string>
+    <string name="hockeyapp_feedback_subject_hint">主题</string>
+    <string name="hockeyapp_feedback_message_hint">内容</string>
+    <string name="hockeyapp_feedback_last_updated_text">最后更新时间: %s</string>
+    <string name="hockeyapp_feedback_attachment_button_text">添加附件</string>
+    <string name="hockeyapp_feedback_send_button_text">发送反馈</string>
+    <string name="hockeyapp_feedback_response_button_text">添加响应</string>
```
Comment (DennisPan) [inline hockeysdk/src/main/res/values-zh/strings.xml]: `回复`

File: hockeysdk/src/main/res/values-zh/strings.xml [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+    <string name="hockeyapp_feedback_refresh_button_text">刷新</string>
+    <string name="hockeyapp_feedback_title">反馈</string>
+    <string name="hockeyapp_feedback_send_generic_error">无法发送反馈，请检查您的输入内容和网络连接后重试。</string>
```
Comment (DennisPan) [inline hockeysdk/src/main/res/values-zh/strings.xml]: `，` --> `。`

File: hockeysdk/src/main/res/values-zh/strings.xml [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+    <string name="hockeyapp_feedback_send_network_error">服务器无响应，请检查您的网络连接后重试</string>
```
Comment (DennisPan) [inline hockeysdk/src/main/res/values-zh/strings.xml]: `，` --> `。`

File: hockeysdk/src/main/res/values-zh/strings.xml [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+    <string name="hockeyapp_feedback_attachment_error">错误</string>
+
+    <!-- Send Feedback Task -->
+    <string name="hockeyapp_feedback_sending_feedback_text">发送反馈中...</string>
+    <string name="hockeyapp_feedback_fetching_feedback_text">获取相关讨论内容中...</string>
+
+    <!-- Login Activity -->
+    <string name="hockeyapp_login_headline_text">请输入您的 HockeyApp 帐户邮件地址和密码来授权对此应用程序的访问。</string>
+    <string name="hockeyapp_login_headline_text_email_only">请输入您的 HockeyApp 帐户邮件地址来授权对此应用程序的访问。</string>
+    <string name="hockeyapp_login_missing_credentials_toast">请填写帐户凭据。</string>
```
Comment (DennisPan) [inline hockeysdk/src/main/res/values-zh/strings.xml]: `凭据` --> `密码`

File: hockeysdk/src/main/res/values-zh/strings.xml [hunk 9]
```diff
@@ -0,0 +1,94 @@
...
+    <string name="hockeyapp_paint_menu_save">保存</string>
+    <string name="hockeyapp_paint_menu_undo">撤消</string>
+    <string name="hockeyapp_paint_menu_clear">清除</string>
+    <string name="hockeyapp_paint_dialog_message">是否保存所画内容？</string>
+    <string name="hockeyapp_paint_dialog_negative_button">放弃</string>
+    <string name="hockeyapp_paint_dialog_positive_button">保存</string>
+    <string name="hockeyapp_paint_dialog_neutral_button">取消</string>
+
+    <!-- Permissions -->
+    <string name="hockeyapp_permission_update_title">需要访问存储</string>
```
Comment (DennisPan) [inline hockeysdk/src/main/res/values-zh/strings.xml]: `需要访问存储权限`

File: hockeysdk/src/main/res/values-zh/strings.xml [hunk 10]
```diff
@@ -0,0 +1,94 @@
...
+    <string name="hockeyapp_permission_dialog_positive_button">重试</string>
+
+    <!-- Dialogs -->
+    <string name="hockeyapp_dialog_positive_button">确定</string>
+    <string name="hockeyapp_dialog_negative_button">取消</string>
+    <string name="hockeyapp_dialog_error_title">错误</string>
+    <string name="hockeyapp_dialog_error_message">发生错误</string>
+
+    <!-- Misc -->
+    <string name="hockeyapp_error_no_network_message">您的设备无法访问 Internet，请在解决网络连接问题后重试。</string>
```
Comment (DennisPan) [inline hockeysdk/src/main/res/values-zh/strings.xml]: `Internet` --> `网络`

`，` --> `。`

File: hockeysdk/src/main/res/values-zh/strings.xml [hunk 11]
```diff
@@ -0,0 +1,94 @@
...
+    <string name="hockeyapp_update_dialog_message">显示关于更新的信息？</string>
+    <string name="hockeyapp_update_dialog_negative_button">忽略</string>
+    <string name="hockeyapp_update_dialog_positive_button">显示</string>
+
+    <!-- Expiry Info -->
+    <string name="hockeyapp_expiry_info_title">版本过期</string>
+    <string name="hockeyapp_expiry_info_text">%s 已经过期，请在 HockeyApp 中检查是否存在更新版本。</string>
+
+    <!-- Feedback Activity -->
+    <string name="hockeyapp_feedback_failed_title">挂起</string>
```
Comment (DennisPan) [inline hockeysdk/src/main/res/values-zh/strings.xml]: This translation looks odd. I was trying to understand the context and then come up with a better one, but checked with @ranterle and the string does not seem to be used, so we may just leave it as it is or remove it.

File: hockeysdk/src/main/res/values-zh/strings.xml (hunk 1 above)
Comment (ranterle) [inline hockeysdk/src/main/res/values-zh/strings.xml]: Not exactly sure what to make of this 😄 - what's the value we should be using?

File: hockeysdk/src/main/res/values-zh/strings.xml (hunk 11 above)
Comment (ranterle) [inline hockeysdk/src/main/res/values-zh/strings.xml]: Leaving this as is.

File: hockeysdk/src/main/res/values-zh/strings.xml (hunk 1 above)
Comment (DennisPan) [inline hockeysdk/src/main/res/values-zh/strings.xml]: haha, let's just change this to

`您是否愿意发送一份匿名报告以便我们修复问题？`

**Comment by ranterle** [comment]
Merged, thanks for the contribution!

**Pull request merged**